extensions/sd_api_pictures/script.py

```diff
@@ -97,19 +152,51 @@ def input_modifier(string):
 
     global params
 
-    if not params['mode'] == 1:  # if not in immersive/interactive mode, do nothing
-        return string
-
-    if triggers_are_in(string):  # if we're in it, check for trigger words
+    if params['mode'] == 1:  # if not in immersive/interactive mode, do nothing
```

Comment:
Right now it should be `#For immersive/interactive mode, use the separate logic`